pkg/k8shandler/cluster.go

```diff
@@ -17,6 +17,8 @@ import (
 var wrongConfig bool
 var nodes map[string][]NodeTypeInterface
 
+var aliasNeeded = true
```

Comment:
Doesn't this mean this could at most happen once for a single ES cluster?  operator starts and its true, during reconciliation we add alias, value is set to false.  How would the operator managing CL and Jaeger alias both clusters?

Reply:
> Doesn't this mean this could at most happen once for a single ES cluster?

Once per restart of the cluster -- if the alias operation was a no-op or successful it'll be set to `false`, otherwise it'll run again.

> How would the operator managing CL and Jaeger alias both clusters?

I don't understand what you're asking here